ajax/libs/color-hash.js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "color-hash.js",
+  "filename": "color-hash.js",
+  "version": "1.0.3",
+  "homepage": "https://zenozeng.github.io/color-hash/demo/",
+  "description": "Generate color based on the given string (using HSL color space and BKDRHash).",
+  "keywords": [
+    "Color Hash",
+    "String"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/zenozeng/color-hash.git",

```

Comment:
Let's use npm auto-update.
Because there are more versions from npm(8) then GitHub tag(6).
The whole config is
```js
  "npmName": "color-hash",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    }
  ],
```
Thank you.